docs/source/AdministratorGuide/Tutorials/basicTutoSetup.rst

```diff
@@ -0,0 +1,731 @@
+.. _tuto_basic_setup:
+
+====================
+Basic Tutorial setup
+====================
+
+Tutorial goal
+=============
+
+The aim of the tutorial is to have a self contained DIRAC setup. You will be guided through the whole installation process both of the server part and the client part.
+By the end of the tutorial, you will have:
+
+* a Configuration service, to serve other servers and clients
+* a ComponentMonitoring service to keep track of other services and agents installed
+* a SystemAdministrator service to manage the DIRAC installation in the future
+* the WebApp, to allow for web interface access
+
+The setup you will have at the end is the base for all the other tutorials.
+
+
+Basic requirements
+==================
+
+We assume that you have at your disposition a fresh SLC6 64bit installation. If you don't, we recommend installing a virtual machine. Instructions for installing SLC6 can be found `here <http://linux.web.cern.ch/linux/scientific6/docs/install.shtml>`_
```

Comment:
Do you want to mention that what follows should also work for other distros from the same family, but it was not actually tried? 
And that it should probably work also for CC7, with minor changes?

Reply:
No, I'd really not give any choice at all for tutorials, keep it as strict as possible 9as per the recommendation of the video :) ). But It' true that I do not think we specify anywhere in the doc for which version of the OS there is support

Reply:
Maybe it could be mentioned in 2 places:
- https://dirac.readthedocs.io/en/latest/UserGuide/GettingStarted/InstallingClient/index.html for the client 
- in https://dirac.readthedocs.io/en/latest/AdministratorGuide/InstallingDIRACService/index.html is specified already, but not at the beginning.